### ISSUE TITLE
Can you improve prose to allow editing jekyll data yaml files

### ISSUE THREAD

**Issue by JayAtCornwallCollege** [prose/prose#830]
i.e. when creating a new file it does not default to a .md extension and does not provide the editor.  I am happy to still configure yaml with _prose.yml

the problem is I need to process the _data folder in jekyll but prose does not easily handle these files

many thanks

**Comment by dereklieu** [comment]
@JayAtCornwallCollege yeah, prose wouldn't default to `.md` on new files unless you put it in `_posts` or `_drafts`. Right now you'd have to save, and reload the page.

**Comment by phillipadsmith** [comment]
@dereklieu Is this an open issue? I'm able to create new .yml and to edit them in prose.io without any issues. The file tree is not refreshed automatically, but -- otherwise -- "works for me."

**Comment by phillipadsmith** [comment]
Bah, @dereklieu, spoke to soon: I see the related ticket about .md files. 

This ticket, however, seems to be specifically about .yml files though...?

**Comment by jonathanduarte** [comment]
Hello, I have a similar problem, I would like to create metadata to easily manipulate parameters in _data and thus manipulate data in html pages within my blog. can you help me? and sorry for my little English.
